helpers/excerpt.php

```diff
@@ -0,0 +1,23 @@
+namespace DustPress;
+
+class Excerpt extends Helper { 
+    public function output() {
+        global $post;
+
+        if ( isset( $this->params->data ) ) {
+            return apply_filters( 'the_excerpt', $this->params->data );
+        } else {
+            if ( isset( $this->params->id ) ) {
+                $post = get_post( $this->params->id );
+            }
+            
+            ob_start();
+            setup_postdata( $post );
+            the_excerpt();
+            wp_reset_postdata();
+            return ob_get_clean(); 
+        }
```

Comment:
No need for the `else` block here, because `return` above stops execution.

```suggestion
        }
        
        if ( isset( $this->params->id ) ) {
            $post = get_post( $this->params->id );
        }
          
        ob_start();
        setup_postdata( $post );
        the_excerpt();
        wp_reset_postdata();
        return ob_get_clean(); 
```